src/index.js

```diff
@@ -16,16 +16,29 @@ console.log('GITHUB_WORKSPACE', GITHUB_WORKSPACE);
 
 const pantheonDeploy = (() => {
 
-    const init = ({
+    const open = ({
         pantheonRepoURL,
         pantheonRepoName,
         machineToken,
         pullRequest
     }) => {
+        checkBranch(pullRequest.head.ref, strictBranchName);
         gitBranch(pantheonRepoURL, pullRequest);
         buildMultiDev(machineToken, pantheonRepoName, pullRequest);
     };
 
+    const checkBranch = (prName, strictBranchName) => {
+        if (prName.length > 11) {
+            core.setFailed("Branch name is too long to create a multidev. Branch names need to be 11 characters or less.");
+            process.abort();
+        } else if (strictBranchName == "strict" && prName.match(/[A-z]*-[0-9]*/)) {
```

Comment:
How do you manage 2 dudes working on the same tickets... So they shoudl be able to use same branch with a differentiator Eg: CHIV-123 & CHIV-123-2